packages/nextjs/app/builders/0xBC428Bb80B1cc3C29164820528819Abf6b20cB88/page.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
```

Comment:
We could remove "use client" from here


Since this is a static page(if you remove js stage from https://github.com/BuidlGuidl/batch8.buidlguidl.com/pull/24/files#r1726164292) you should be able to make this an server component.

In NextJs, server component comes with alot of benefits like SSR [etc](https://nextjs.org/docs/app/building-your-application/rendering/server-components#benefits-of-server-rendering)